### ISSUE TITLE
grpc rebuilding for Electron failed.

### ISSUE THREAD

**Issue by eouia** [grpc/grpc-node#507]
### Problem description
grpc rebuilding failed.

### Reproduction steps
```
npm install --build-from-source grpc
```
### Environment
 - OS name, version and architecture: Raspbian Stretch (Raspberry Pi 3 B+)
 - Node version: v9.11.2
 - Node installation method : nvm
 - Package name and version : grpc-node@latest (?)

### Additional context
I want to use this module with Electron, but failed to rebuild. Here is error log.
```
...
  CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/resolver/dns/native/dns_resolver.o
  CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/client_channel/resolver/sockaddr/sockaddr_resolver.o
make: *** No rule to make target 'Release/obj.target/grpc/deps/grpc/src/cpp/ext/filters/census/grpc_context.o', needed by 'Release/obj.target/libgrpc.a'.  Stop.
make: Leaving directory '/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:180:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:12)
gyp ERR! System Linux 4.14.50-v7+
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--library=static_library" "--module=/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/src/node/extension_binary/node-v59-linux-arm-glibc/grpc_node.node" "--module_name=grpc_node" "--module_path=/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/src/node/extension_binary/node-v59-linux-arm-glibc" "--napi_version=3" "--node_abi_napi=napi"
gyp ERR! cwd /home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc
gyp ERR! node -v v9.11.2
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --library=static_library --module=/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/src/node/extension_binary/node-v59-linux-arm-glibc/grpc_node.node --module_name=grpc_node --module_path=/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/src/node/extension_binary/node-v59-linux-arm-glibc --napi_version=3 --node_abi_napi=napi' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:180:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:936:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:220:5)
node-pre-gyp ERR! System Linux 4.14.50-v7+
node-pre-gyp ERR! command "/usr/bin/node" "/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build" "--library=static_library"
node-pre-gyp ERR! cwd /home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc
node-pre-gyp ERR! node -v v9.11.2
node-pre-gyp ERR! node-pre-gyp -v v0.10.3
node-pre-gyp ERR! not ok
Failed to execute '/usr/bin/node /usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --library=static_library --module=/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/src/node/extension_binary/node-v59-linux-arm-glibc/grpc_node.node --module_name=grpc_node --module_path=/home/pi/MagicMirror/modules/MMM-AssistantMk2/node_modules/grpc/src/node/extension_binary/node-v59-linux-arm-glibc --napi_version=3 --node_abi_napi=napi' (1)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! grpc@1.14.1 install: `node-pre-gyp install --fallback-to-build --library=static_library`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the grpc@1.14.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2018-08-20T08_28_12_184Z-debug.log
```

**Comment by nicolasnoble** [comment]
Yeah, we've found this one already, and it's getting fixed. You can downgrade to 1.13 if you can't wait for the new package to be published. cc @murgatroid99.

**Comment by murgatroid99** [comment]
Right. The fix for this is in #501. We'll publish a new version soon.

**Comment by eouia** [comment]
Ok. I'll wait. thanks. I'll close this issue.
